fem/geom.cpp

```diff
@@ -283,6 +309,67 @@ bool Geometry::CheckPoint(int GeomType, const IntegrationPoint &ip)
    return true;
 }
 
+bool Geometry::CheckPoint(int GeomType, const IntegrationPoint &ip, double eps)
```

Comment:
There was only one other use of ``Geometry::CheckPoint()`` in ``mfem``, and it had a comment about considering to add an epsilon.  Perhaps that case would also benefit from this new function.

See:  ``FiniteElementSpace::GetLocalDerefinementMatrices()``

Reply:
@jakubcerveny, can you comment on this?